src/frame.js

```diff
@@ -0,0 +1,29 @@
+function Frame (seqNum) {
+        this.rolls = [];
+        this.frames = [];
+        this.seqNum = seqNum || 0;
+
+    Frame.prototype.totalRolls = function() {
+        // if (this.rolls.length === 3) {
+        //     this.rolls.splice(0, this.rolls.length)
+        // }
+        return (this.rolls.length);
+    };
+    Frame.prototype.storeRoll = function(roll) {
+        return (this.rolls.push(roll));
+    };
+
+    Frame.prototype.newFrame = function(updateFrame) {
+        if (this.rolls.length === 2) {
+            this.rolls.splice(0, this.rolls.length)
```

Comment:
as you have your totalRolls function currently returning the length of your rolls array, you could call this here to keep your code Dry.